dist.sh

```diff
@@ -70,16 +78,27 @@ for platform in $platforms; do
 
     cp -r $template_dir/* $dist_path/
 
-    cp $core_dir/build/libs/*.jar $dist_path/workcraft.jar
+    # resource_dir is simply the distribution root. OS X needs special care.
+    resource_dir=$dist_path
```

Comment:
I do not see how to make it more readable without duplicating the code for OSX branch...

Maybe introduce `src_dir` and `dst_dir` for the copy operation, like this (roughly):

    ...
    if [ "$platform" = "osx" ]; then
        contents_dir=$dist_dir/Contents
        # Update Info.plist with version
        echo "$(sed -i "s/__VERSION__/$version/" $contents_dir/Info.plist)"
        dst_dir=$contents_dir/Resources
    else
       dst_dir=$dist_dir
    fi
    src_dir=$core_dir/build/libs

    cp $src_dir/*.jar $dst_dir/workcraft.jar
    ...